cmd/afsPmf/afsPmf.go

```diff
@@ -9,8 +9,8 @@ import (
 	"log"
 )
 
-func afsPmf(vcfFile string, outFile string, unPolarized bool) {
-	g, err := popgen.VcfToAfs(vcfFile, !unPolarized) //VcfToAfs is written in terms of polarized, so this is inverted here.
+func afsPmf(vcfFile string, outFile string, s popgen.McmcSettings) {
+	g, err := popgen.VcfToAfs(vcfFile, s) //VcfToAfs is written in terms of polarized, so this is inverted here.
```

Comment:
Do we need to pass the whole settings struct?  I am guessing that we are only using a couple of those settings, but for a programmer it might be hard to know what ones need to be set.  Do we only use unPolarized?  Maybe we can just pass in s.UnPolarized?